backend/controller/uwfinancepurchases.controller.js

```diff
@@ -64,7 +73,8 @@ const updateApprovalsUWFinancePurchaseController = async (req, res) => {
     const { ticket_data, approval_type, identifier } = req.body
 
     const { isAdmin, isTeamCaptain, isDirector } = await getAuthRoles(
-        identifier
+        identifier,
+        ticket_data?.reporter_id
```

Comment:
actually, we shouldn't even be using getAuthRoles cause it's moved to middleware